vpr/src/base/vpr_types.h

```diff
@@ -1017,6 +1017,7 @@ enum e_routing_failure_predictor {
 };
 enum e_routing_budgets_algorithm {
```

Comment:
Add comment here on what this controls. Either describe the options or refer back to command option help.